docs/DeveloperGuide.md

```diff
@@ -1060,6 +1058,31 @@ testers are expected to do more *exploratory* testing.
     (where incomplete details are given for the applicant being added).
     Expected: Error messages displaying the cause of error is shown in the result display.  
 
+### Editing an applicant
+
+1. Editing an applicant while all applicants are being shown
+
+  1. Prerequisites: List all applicants using the `list` command. Multiple applicants in the list.
```

Comment:
I think `list` is not that necessary a prerequisite for `edit` since a user can edit after any command is executed?

Reply:
I think the `list` is to ensure that all scenario of "all applicants are being shown" is in play.